training/distributed_training/pytorch/model_parallel/gpt-j/01_train_gptj_smp_notebook.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            "prescaled_batch": 0,  # was 1
</pre><p>What does it mean that the prescaled batch is 0?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
the prescaled_batch parameter is set to False